src/test/java/hudson/plugins/analysis/collector/workflow/WorkflowCompatibilityTest.java

```diff
@@ -0,0 +1,86 @@
+package hudson.plugins.analysis.collector.workflow;
+
+import static org.junit.Assert.*;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import hudson.model.Result;
+
+import hudson.plugins.analysis.collector.AnalysisResultAction;
+
+import hudson.FilePath;
+
+/**
+ * Test workflow compatibility.
+ */
+public class WorkflowCompatibilityTest {
+
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
+
+    /**
+     * General collector test using Findbugsas example.
+     */
+    @Test
+    @Ignore("This test requires FindBugs 4.62 or later. Ignoring until it is released")
+    public void collectorPublisherWorkflowStep() throws Exception {
+        WorkflowJob job = j.jenkins.createProject(WorkflowJob.class, "wf");
+        FilePath workspace = j.jenkins.getWorkspaceFor(job);
+        FilePath report = workspace.child("target").child("findbugs.xml");
+        report.copyFrom(WorkflowCompatibilityTest.class.getResourceAsStream("/findbugs-native.xml"));
+        job.setDefinition(new CpsFlowDefinition(
+        "node {\n" +
+        "  step([$class: 'FindBugsPublisher'])\n" +
+        "  step([$class: 'AnalysisPublisher', isFindBugsActivated: true])\n" +
```

Comment:
All these `…Activated` properties default to true, so it should not be necessary to specify them.